designer/src/com/android/tools/idea/common/property/editors/EnumEditor.java

```diff
@@ -35,7 +35,6 @@
 import com.intellij.ui.JBColor;
 import com.intellij.ui.scale.JBUIScale;
 import com.intellij.util.ui.JBUI;
-import com.sun.java.swing.plaf.windows.WindowsComboBoxUI;
```

Comment:
If you change this line to  
```
import com.sun.java.swing.plaf.windows.BasicComboBoxUI;  
```  
You won't have to delete the ComboBoxUI logic below.